tests/integration/standard/test_cluster.py

```diff
@@ -1332,7 +1332,7 @@ def test_address_translator_basic(self):
         lh_ad = LocalHostAdressTranslator({'127.0.0.1': '127.0.0.1', '127.0.0.2': '127.0.0.1', '127.0.0.3': '127.0.0.1'})
         c = TestCluster(address_translator=lh_ad)
         c.connect()
-        self.assertEqual(len(c.metadata.all_hosts()), 1)
+        self.assertEqual(len(set([h.address for h in c.metadata.all_hosts()])), 1)
```

Comment:
Why is this change needed? Does it mean that `all_hosts()` can return the same host more than once? If so, then isn't that a bug?

The test `Validates AddressTranslator invocation by ensuring that only meta data associated with single host is populated` - see the comment. I don't think it does after this change.

Reply:
removed this change